openevsehttp/__main__.py

```diff
@@ -309,27 +313,37 @@ async def get_override(self) -> None:
     async def set_override(
         self,
         state: str,
-        charge_current: int,
-        max_current: int,
-        energy_limit: int,
-        time_limit: int,
+        charge_current: int | None = None,
+        max_current: int | None = None,
+        energy_limit: int | None = None,
+        time_limit: int | None = None,
         auto_release: bool = True,
```

Comment:
Should `auto_release` be true by default @firstof9? I think that is a bit against expectations when pausing charging. I wasn't even aware this was an option on the API until I saw this.